rename_in_folders

```diff
@@ -0,0 +1,17 @@
+import pathlib
+import re
+
+rom_root = pathlib.Path('/my/rom/root/folder/')
```

Comment:
Thank you very much for contributing! Perhaps this could be an input, so the user can input their own specific file path?